mne/viz/tests/test_raw.py

```diff
@@ -189,6 +193,16 @@ def test_plot_raw():
         plt.close('all')
 
 
+@testing.requires_testing_data
+def test_plot_ref_meg():
+    """Test plotting ref_meg."""
+    import matplotlib.pyplot as plt
+    raw_ctf = read_raw_ctf(ctf_fname_continuous).crop(0, 1).load_data()
+    raw_ctf.plot()
+    plt.close('all')
```

Comment:
We usually do try include close commands in viz tests. I think it's mostly a thing to keep mpl memory use down. I moved this test because it seemed to make more sense here, although it did also speed things up a little bit.